src/components/Trip-Planner/Suggestions/List-Items.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import ListSubheader from "@material-ui/core/ListSubheader";
+import DashboardIcon from "@material-ui/icons/Dashboard";
+import ShoppingCartIcon from "@material-ui/icons/ShoppingCart";
+import PeopleIcon from "@material-ui/icons/People";
+import BarChartIcon from "@material-ui/icons/BarChart";
+import LayersIcon from "@material-ui/icons/Layers";
+import AssignmentIcon from "@material-ui/icons/Assignment";
+// import nearbyPlaces from "../../../../sample-data/nearby-places.js";
+
+//this did not work how expected
+const NearbyListItems = ({ listofplaces }) => {
+	return (
+		<>
+			{listofplaces.map((p) => {
+				<ListItem button>
+					<ListItemIcon>
+						<DashboardIcon />
+					</ListItemIcon>
+					<ListItemText primary={p.name} />
+				</ListItem>;
+			})}
+		</>
+	);
+};
+
+export default NearbyListItems;
```

Comment:
Just a naming convention thing but maybe rename your files to have the same name as the component you are exporting? For ease of use!